capstone/src/main/webapp/lesson.html

```diff
@@ -121,15 +122,15 @@ <h5 class="modal-title" id="ModalCenterTitle">Content</h5>
                 </div>
                 <div class="modal-body">
                         <p> Content Information: </p>
-                        <input type="text" id="title" placeholder="Title" required>
-                        <input type="text" id="description" placeholder="Description" required>  
+                        <input type="text" id="content-title" placeholder="Title" required>
+                        <input type="text" id="content-description" placeholder="Description" required>  
                         <input type="text" id="content" placeholder="Content" required>
-                        <input type="text" id="urls" placeholder="URL(s)" required>
+                        <input type="text" id="content-urls" placeholder="URL(s)" required>
```

Comment:
These should work fine as classes probably? At least for title and description, it seems like setting a nice generic class would allow you to keep the targetting code the same as well as potentially use common CSS across all of them